pvlib/tracking.py

```diff
@@ -44,12 +44,17 @@ def singleaxis(apparent_zenith, apparent_azimuth,
         A value denoting the compass direction along which the axis of
         rotation lies. Measured in decimal degrees east of north.
 
-    max_angle : float, default 90
+    max_angle : float or tuple, default 90
         A value denoting the maximum rotation angle, in decimal degrees,
         of the one-axis tracker from its horizontal position (horizontal
-        if axis_tilt = 0). A max_angle of 90 degrees allows the tracker
-        to rotate to a vertical position to point the panel towards a
-        horizon. max_angle of 180 degrees allows for full rotation.
+        if axis_tilt = 0). If a float is provided, it represents the maximum
+        rotation angle, and the minimum rotation angle is assumed to be the
+        opposite of the maximum angle. If a tuple of (min_angle, max_angle)
+        is provided, it represents both the minimum and maximum rotation angles.
+        
```

Comment:
I feel like we need to explain the orientation that distinguishes `min_angle` from `max_angle`. Suggest adding a new paragraph like:

`A rotation to ``'max_angle'`` is a counter-clockwise rotation about the y-axis of the tracker coordinate system. For example, for a tracker with ``'axis_azimuth'`` oriented to the south, a rotation to ``'max_angle'`` is towards the west, and a rotation toward ``'min_angle'`` is in the opposite direction, toward the east.

Reply:
Thank you for this input! As you suggested, I have added this paragraph.